### DISCUSSION TITLE
How can I get information about meeting participants when the meeting finished?

### DISCUSSION THREAD

**Discussion by genius987** [mynaparrot/plugNmeet-server#547] [Q&A]
I saw the following picture in the services offered by MyParrot on PlugNmeet. Could you explain to me which API and an example of source to extract all the information? On BigBlueButton I imported the generated XML file and processed it, how does it work on PlugNmeet?

<img width="1012" alt="viewreport" src="https://github.com/user-attachments/assets/e8bbe423-5e26-4624-9e78-c714faeec75f">

**Comment by jibon57** [comment]
Hi,

You can follow: https://www.plugnmeet.org/docs/api/analytics/fetch

Thanks

**Reply by genius987** [reply]
Thanks jobon57 I tried and now I fetch the information about the room.

I have a question, i don't see the origin of the room, for example if i started the room from moodle partion i would like to read the original domain from which the room started. For example the room belong at moodle.example.com Is it possible? My goal is create the reports for single course.
